src/messagecommands/SendUpdate.ts

```diff
@@ -0,0 +1,36 @@
+import { Message, Permissions, MessageEmbed, TextChannel } from 'discord.js';
+import { Bot } from '../Bot';
+import { MessageCommand } from './MessageCommand';
+import config from '../../config.json';
+import { updateChannels } from '../slashcommands/Update';
+
+export class SendUpdate implements MessageCommand {
```

Comment:
Technically, this doesn't need to be a message command. If you set the default permissions to one user, it disables the command for everyone else, making it not even show up with the / menu. See https://discordjs.guide/interactions/slash-command-permissions.html#user-permissions
We don't use the .set function, but the format of permissions: [{}] can be used in the registerData object next to name and description.
However, currently since it doesn't delete and re-register all slash commands, it won't update slash command permissions after you've set them once, so I'll allow this for now. In the future, once I've fixed that (I'll just make it so that it deletes and re-registers any command with permissions, since we have so few of them) I'd like this to be moved over to a slash command.

Reply:
Actually, looking into it, it seems that working with slash command permissions application-wide is a whole ordeal. Keep it as a message command for now.